.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.25
+  - jruby-9.0.5.0
```

Comment:
>  just figure that if the tests pass that's a bonus

Ok, makes sense to run both... Just don't want us to block a PR because it fails on JRuby 9K (yet)